docs/archive.md

```diff
@@ -0,0 +1,26 @@
+# Data Archiving
+
+When the crawls are complete, the data is archived to several places:
+
+- Internet Archive (sql)
```

Comment:
Right, that's my pedantic point: there is no "SQL" as a data format. MySQL dump is a special flavor. That said, I am nitpicking.